pivotal-import/lib.py

```diff
@@ -471,12 +503,32 @@ def parse_date_time(d: str):
 
 
 ### Utility functions
+
+
+def dissoc(dict, key_to_remove):
+    """Return a copy of `dict` with `key_to_remove` absent."""
+    d = deepcopy(dict)
```

Comment:
I think `dict.copy` is sufficient? Deep copy is useful for nested collections.

Reply:
Makes sense. As a utility function, I didn't want any of us to get surprised by nested dicts not being supported by this function. I think they're functionally equivalent if it's a shallow dict, so going to keep in place for now and can use the less expensive call if performance requires it.